output_template/python/usb_camera_demo.py

```diff
@@ -283,6 +284,61 @@ def run_sementic_segmentation(config):
         m1.show()
 
 
+def run_keypoint_detection(config):
+    global nn
+    camera_width = 320
+    camera_height = 240
+    window_name = "Keypoint Detection Demo"
+
+    input_width = config.IMAGE_SIZE[1]
+    input_height = config.IMAGE_SIZE[0]
+
+    vc = init_camera(camera_width, camera_height)
+
+    pool = Pool(processes=1, initializer=nn.init)
+    result = None
+    fps = 1.0
+
+    q_save = Queue()
+    q_show = Queue()
+
+    grabbed, camera_img = vc.read()
+    if not grabbed:
+        print("Frame is empty")
+
+    q_show.put(camera_img.copy())
+    input_img = camera_img.copy()
+
+    while True:
+        m1 = MyTime("1 loop of while(1) of main()")
+        pool_result = pool.apply_async(_run_inference, (input_img,))
+        is_first = True
+        while True:
+            grabbed, camera_img = vc.read()
+            if is_first:
+                input_img = camera_img.copy()
+                is_first = False
+            q_save.put(camera_img.copy())
+            if not q_show.empty():
+                window_img = q_show.get()
+                drawed_img = window_img
+                if result is not None:
+
+                    drawed_img = visualize_keypoint_detection(window_img, result[0], (input_height, input_width))
+                    drawed_img = add_fps(drawed_img, fps)
+
+                cv2.imshow(window_name, drawed_img)
+                key = cv2.waitKey(2)  # Wait for 2ms
```

Comment:
Shouldn't we use Ctrl-C for abort?

Reply:
by screen do you mean X11? This still seem like a bad design to me, as the UI and logic is too tightly mixed.

Reply:
I have added a TODO comment for this.